admin/class-flizpay-admin.php

```diff
@@ -207,6 +204,16 @@ public function load_form_fields()
 				),
 				'desc_tip' => true,
 			),
+			'flizpay_sentry_enabled' => array(

```

Comment:
You need to initialize, obtain and save this option on the gateway class in the function that handles the saving of the options. It won't be saved automatically